internal/quickstart/container.go

```diff
@@ -57,7 +65,7 @@ func (m ContainerModel) Update(msg tea.Msg) (tea.Model, tea.Cmd) {
 			cmd = tea.Quit
 		case key.Matches(msg, keys.Back):
 			// if showing SDK instructions, let the user go back to choose a different SDK
-			if m.currentStep == 2 {
+			if m.currentStep == stepShowSDKInstructions {
```

Comment:
2 is wrong. It should be 3 which is the show SDK instructions page.

Reply:
yay the iotas were nice for this